### ISSUE TITLE
New license request: BSD-Advertising-Acknowledgement [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by richardfontana** [spdx/license-list-XML#1645]
**1.** License Name: None
**2.** Short identifier: BSD-4-Clause-Attribution
**3.** License Author or steward: Unknown
**4.** Comments: This license appears to be a legacy BSD-derivative license that I believe would generally be assumed to be FOSS. It apparently covers some portion of a project (xlrd) that is packaged in PyPI and Fedora. As far as I can tell it is not reflected in the current SPDX license list. It is like BSD-3-Clause-Attribution but with the BSD-4-Clause advertising clause added. It is somewhat similar to the first part of the OpenSSL license but with clause 5 omitted.
**5.** Standard License Header: 
**6.** License Request Url: http://tools.spdx.org/app/license_requests/169
**7.** URL(s): https://github.com/python-excel/xlrd/blob/master/LICENSE#L33  
https://github.com/python-excel/xlwt/blob/master/LICENSE#L44
**8.** OSI Status: Not Submitted
**9.** Example Projects: https://github.com/python-excel/xlrd  
https://github.com/python-excel/xlwt/

**Comment by richardfontana** [comment]
The suggested short identifier "BSD-4-Clause-Attribution" was based on the current "BSD-3-Clause-Attribution".

**Comment by Xavierj2800** [comment]
> **1.** License Name: None
> **2.** Short identifier: BSD-4-Clause-Attribution
> **3.** License Author or steward: Unknown
> **4.** Comments: This license appears to be a legacy BSD-derivative license that I believe would generally be assumed to be FOSS. It apparently covers some portion of a project (xlrd) that is packaged in PyPI and Fedora. As far as I can tell it is not reflected in the current SPDX license list. It is like BSD-3-Clause-Attribution but with the BSD-4-Clause advertising clause added. It is somewhat similar to the first part of the OpenSSL license but with clause 5 omitted.
> **5.** Standard License Header:
> **6.** License Request Url: http://tools.spdx.org/app/license_requests/169
> **7.** URL(s): https://github.com/python-excel/xlrd/blob/master/LICENSE#L33
> https://github.com/python-excel/xlwt/blob/master/LICENSE#L44
> **8.** OSI Status: Not Submitted
> **9.** Example Projects: https://github.com/python-excel/xlrd
> https://github.com/python-excel/[xcs3q3vc here go 2.txt](https://github.com/spdx/license-list-XML/files/9654688/xcs3q3vc.here.go.2.txt)

**Comment by swinslow** [comment]
+1 to add; despite the attribution clause, this is still a FOSS license and has widespread use based on inclusion in Fedora.

As far as name and identifier, I'm not entirely convinced on calling it "BSD-4-Clause-Attribution". For "BSD-3-Clause-Attribution", it looks like that was BSD-3-Clause plus an attribution clause. This appears to be BSD-4-Clause, plus attribution and _minus_ the "no endorsement" 4th clause.

Maybe `BSD-Advertising-Acknowledgment`?

Also a reminder to the general public, please stop drafting new BSD-style licenses  ;-)

**Comment by Pizza-Ria** [comment]
Definite +1 on @swinslow's reminder to the general public although this seems to be a few years old so, hopefully, this is not a continuing issue. I'm OK with @swinslow's suggested name.  I assume this language would be excised "Based on the the Java version: Andrew Khan Copyright (c) 2000." I'm assuming this is in reference to the code rather than the license.  Otherwise +1 to add.

**Comment by swinslow** [comment]
# License Inclusion Decision

## Decision:

- [X] approved
- [ ] not approved

## License full name

BSD Advertising Acknowledgement License

## Short ID

BSD-Advertising-Acknowledgment

## XML markup

Add replaceable templating for author's name in clauses 3 and 4, and disclaimer, in similar manner as other BSD licenses

## Additional rationale or notes on decision:

Agree with @Pizza-Ria on omitting the "Based on the..." language from the template for this one!

**Comment by swinslow** [comment]
Ooof... so I just realized that I made a typo with different spellings for "acknowledgment" vs. "acknowledgement" ("e" after "g") for the ID vs. name in the [inclusion decision above](https://github.com/spdx/license-list-XML/issues/1645#issuecomment-1372442669).

Looking at licenses on the existing license list, it looks like "with the e" is the format we've used previously: see [zlib-acknowledgement](https://spdx.org/licenses/zlib-acknowledgement.html).

Given that, since we need to resolve this (and I wouldn't particularly want to have different spellings for name vs. ID), I'm going to revise this to "with the e" for both, to match zlib-acknowledgement. British spelling wins, I suppose  ;)

I'm going to update this issue and the PR at #1818 to fix this before merging #1818.
